clients/docs/includes/clients-checkout.rst

```diff
@@ -1,9 +1,7 @@
 Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__
-GitHub repository and check out the
-:litwithvars:`|release|-post` branch.
+GitHub repository and check out the branch.
```

Comment:
suggest that we say to check out the `latest` branch now that we're keeping that branch up to date (and below `git checkout latest`